geoportailv3/__init__.py

```diff
@@ -28,6 +28,18 @@ def main(global_config, **settings):
         authentication_policy=create_authentication(settings)
     )
 
+    # overwrite print routes
+    config.add_route(
+        "lux_printproxy_report_create",
+        "/printproxy/report.{format}",
+        request_method="POST"
+    )
+    config.add_route(
+        "lux_printproxy_report_get",
+        "/printproxy/report/{ref}",
+        request_method="GET"
```

Comment:
I just tried to cancel a print request in the geoportailv3 project, and it does not work. I get a 404 from the mapfish print service:

```
2015-06-22 11:44:26,464 INFO  [c2cgeoportal.views.proxy][Dummy-4] Send query to URL:
http://localhost:8080/print-elemoine/print/geoportailv3/cancel/cd3020f7-a6e5-4665-a24d-b2767d001ef4@9a0c91e4-95ae-4495-9cf7-8378f802ab7f?.
2015-06-22 11:44:26,468 ERROR [c2cgeoportal.views.proxy][Dummy-4] Error 'Not Found' in response of URL:
http://localhost:8080/print-elemoine/print/geoportailv3/cancel/cd3020f7-a6e5-4665-a24d-b2767d001ef4@9a0c91e4-95ae-4495-9cf7-8378f802ab7f?.
2015-06-22 11:44:26,468 ERROR [c2cgeoportal.views.proxy][Dummy-4] Status: 404
2015-06-22 11:44:26,468 ERROR [c2cgeoportal.views.proxy][Dummy-4] Method: DELETE
2015-06-22 11:44:26,468 ERROR [c2cgeoportal.views.proxy][Dummy-4] --- With headers ---
Origin: http://localhost:5000
Accept-Language: en-US,en;q=0.8,fr;q=0.6
Accept-Encoding: gzip, deflate, sdch
Host: localhost:5000
Accept: application/json, text/plain, */*
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.81 Safari/537.36
Connection: keep-alive
Referer: http://localhost:5000/theme/main?version=3&zoom=15&X=680824&Y=6378491&debug&crosshair=false
Pragma: no-cache
Cache-Control: no-cache
2015-06-22 11:44:26,468 ERROR [c2cgeoportal.views.proxy][Dummy-4] --- Return content ---
2015-06-22 11:44:26,468 ERROR [c2cgeoportal.views.proxy][Dummy-4] <html><head><title>Apache Tomcat/7.0.52 (Ubuntu) - Error report</title><style><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color : #525D76;}--></style> </head><body><h1>HTTP Status 404 - </h1><HR size="1" noshade="noshade"><p><b>type</b> Status report</p><p><b>message</b> <u></u></p><p><b>description</b> <u>The requested resource is not available.</u></p><HR size="1" noshade="noshade"><h3>Apache Tomcat/7.0.52 (Ubuntu)</h3></body></html>
```